snap/snapcraft.yaml

```diff
@@ -12,6 +12,21 @@ description: |-
 
 confinement: strict
 
+slots:
+  ovn-api:
+    interface: content
+    source:
+      read:
+        - "$SNAP_COMMON/data/pki/client-cert.pem"
+        - "$SNAP_COMMON/data/pki/client-privkey.pem"
+        - "$SNAP_COMMON/data/pki/cacert.pem"
+        - "$SNAP_COMMON/data/ovn.env"
+  ovn-chassis:
```

Comment:
minor note about a slight disconnect in the name of this slot promising access to both OVS and OVN sockets, and that it in this point in time only provides the OVS sockets.

I assume this is because only the OVS sockets are needed. If more is needed later we can always extend/move things.

Reply:
If you think it would be more correct, I can rename this to `ovs-chassis` and `ovn-chassis` can be introduced later if it becomes necessary.

Reply:
No need to change now, just wanted to make a note of it.

Reply:
No need to change now, just wanted to make a note of it.